ci-automation/push_pkgs.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# >>> This file is supposed to be SOURCED from the repository ROOT. <<<
+#
+# push_packages() should be called w/ the positional INPUT parameters below.
+
+# OS image binary packages publisher automation stub.
```

Comment:
Why stub?

Reply:
Because it's just glue between the SDK container and the CI automation. CI must implement the other half.